go/tasks/plugins/webapi/bigquery/plugin.go

```diff
@@ -291,6 +307,14 @@ func (p Plugin) Status(ctx context.Context, tCtx webapi.StatusContext) (phase co
 	return core.PhaseInfoUndefined, pluginErrors.Errorf(pluginsCore.SystemErrorCode, "unknown execution phase [%v].", resource.Status.State)
 }
 
+func constructOutputLocation(job *bigquery.Job) string {
+	if job == nil || job.Configuration == nil || job.Configuration.Query == nil || job.Configuration.Query.DestinationTable == nil {
+		return ""
+	}
+	dst := job.Configuration.Query.DestinationTable
+	return fmt.Sprintf("bq://%v:%v.%v", dst.ProjectId, dst.DatasetId, dst.TableId)
```

Comment:
one more thing @pingsutw - can you debug log this?

Reply:
updated it